lib/ruby-mpd/plugins/information.rb

```diff
@@ -11,14 +11,12 @@ def clearerror
         send_command :clearerror
       end
 
-      # Get the currently playing song
+      # Get the currently playing song data
       #
-      # @return [MPD::Song]
-      # @return [nil] if there is no song playing
-      def current_song
-        hash = send_command :currentsong
-        # if there is no current song (we get true, then return nil)
-        hash.is_a?(TrueClass) ? nil : Song.new(self, hash)
+      # @return [Hash]
+      # @return [TrueClass] if there is no song playing
+      def currentsong
```

Comment:
This can't be modified because we break the previous API. Just call `send_command` inside the new method. I'll leave a deprecation notice for this command later